content/source/docs/enterprise/api/ssh-keys.html.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: enterprise2
+page_title: "SSH Keys - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-ssh-keys"
+---
+
+# SSH Keys
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+The `ssh-key` object represents an SSH key which includes a name and the SSH private key. This object is used to clone Terraform modules at the organization level, and allows multiple keys to be added for the organization. You can also easily use them in any [workspace](./workspaces.html#assign-an-ssh-key-to-a-workspace) that clones modules from a Git server.
+
+## List SSH Keys
+
+List all the SSH Keys for a given organization
+
+| Method | Path           |
+| :----- | :------------- |
+| GET | /organizations/:organization_name/ssh-keys |
+
+### Parameters
+
+- `:organization_name` (`string: <required>`) - specifies the organization name where the SSH Keys are defined
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  https://app.terraform.io/api/v2/organizations/my-organization/ssh-keys
+```
+
+### Sample Response
+
+```json
+{
+  "data": [
+    {
+      "attributes": {
+        "name": "SSH Key"
+      },
+      "id": "sshkey-GxrePWre1Ezug7aM",
+      "links": {
+        "self": "/api/v2/ssh-keys/sshkey-GxrePWre1Ezug7aM"
+      },
+      "type": "ssh-keys"
+    }
+  ]
+}
+```
+
+
+## Get an SSH Key
```

Comment:
...Is there any purpose to ever doing this action? I guess if you have an ID and need to map it to a name.

This page could use some clarification that we never allow you to obtain the actual private key text, but I won't hold up the review on that and am OK with merging without it for now.

Reply:
@nfagerlund I just went ahead and documented it, but yeah, it ain't entirely useful since you're just getting a name. A hint about not returning the private key text would be great. If you have a bit of text you would prefer I can tack that on separately.